pkg/apis/apps/v1alpha1/apimanager_types.go

```diff
@@ -188,6 +238,14 @@ type ZyncSpec struct {
 	Image *string `json:"image,omitempty"`
 	// +optional
 	PostgreSQLImage *string `json:"postgreSQLImage,omitempty"`
+
+	// +optional
+	ZyncAppSpec *ZyncAppSpec `json:"zyncAppSpec,omitempty"`
+}
+
+type ZyncAppSpec struct {
```

Comment:
There's no DeploymentConfig name "zync-app" but I could not choose ZyncSpec because ZyncSpec already exists as a section that contains this field.